### PR TITLE
Unify icon styling

### PR DESCRIPTION
**Pull request by da-Kai**

### CODE REVIEW

File: ui/src/app/shared/type/widget.ts [hunk 1]
```diff
@@ -44,6 +44,11 @@ export type Icon = {
     name: string;
 };
 
+export type ImageIcon = {
+    src: string;
+    large: boolean;
```
Comment (lukasrgr) [inline ui/src/app/shared/type/widget.ts:49]: not sure what you want to provide here, shouldnt this be `size`?

File: ui/src/global.scss [hunk 2]
```diff
@@ -396,3 +396,17 @@ ion-modal.full-width {
...
+    margin-top: 12px;
+    margin-bottom: 12px;
+    --border-radius: 0;
+    max-height: 24px;
+    max-width: 24px;
+}
+
+ion-avatar.iconify.icon-large {
+    max-height: 32px;
+    max-width: 32px;
```
Comment (lukasrgr) [inline ui/src/global.scss:411]: im always worried about fixed ratios? are we sure this works out for all display res?

**Review by lukasrgr — CHANGES_REQUESTED** [review]

File: ui/src/global.scss (hunk 2 above)
Comment (da-Kai) [reply ui/src/global.scss:411]: i copied these values from ion-icon

**Review by lukasrgr — APPROVED** [review]
🚀

**Review by sfeilmeier — APPROVED** [review]

**Pull request merged**